src/Acuminator/Acuminator.Tests/Sources/Dac/PX1031 and PX1032/Diagnostics/DacExtensionWithMethodsUsage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using PX.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Acuminator.Tests.Sources
+{
+    [Serializable]
+    public class DacExtensionWithMethodsUsage : PXCacheExtension<IIGPOALCLandedCost>
```

Comment:
@metallmonster, please add more test cases: a delegate invocation, a constructor invocation,  You can add them here in other properties.

Reply:
@SENya1990 ok